platform/x11/os_x11.cpp

```diff
@@ -1330,6 +1330,7 @@ void OS_X11::process_xevents() {
 				Ref<InputEventMouseButton> mb;
 				mb.instance();
 
+				mb->set_factor(1.0);
```

Comment:
That should be the default value for InputEventMouseButton IMO, so better fix it in the class's definition than each time it's instanced.